### PR TITLE
Feature/m100

### PR DESCRIPTION
**Pull request by zx70**

### CODE REVIEW

**Comment by suborb** [comment]
Target number 100!

File: libsrc/target/m100/stdio/generic_console.asm [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+generic_console_set_paper:
+generic_console_set_attribute:
+	ret
+
+
+generic_console_set_ink:
+	ret
+
+generic_console_scrollup:
+	call $45ED	; SCROLL_ALTLCD
```
Comment (suborb) [inline libsrc/target/m100/stdio/generic_console.asm]: scrollup needs to preserve bc,de - does SCROLL_ALTLCD do that?

Comment (zx70) [reply libsrc/target/m100/stdio/generic_console.asm]: Looks like a valid hint.  I'm still improving the whole thing,  e.g. locations vary depending on the ROM versions, etc..

**Comment by zx70** [comment]
https://github.com/z88dk/z88dk/commit/b3029f1422cbf3638cc74e5a331ad95ec4fb0a6c

rpn.c is now reasonably ok,  tic.c is still bad when scrolling.

**Pull request merged**